extensions/2.0/Vendor/EXT_mesh_features/README.md

```diff
@@ -616,13 +617,69 @@ Texture filtering must be `9728` (NEAREST), `9729` (LINEAR), or undefined, for a
+| `"UINT16"`               | `5123`                   | `"SINGLE"`      | `"SCALAR"`             |
+| ...                      | ...                      | ...             | ...                    |
+| `"UINT32"`               | `5125` <sup>2</sup>      | `"SINGLE"`      | `"SCALAR"`             |
+| ...                      | ...                      | ...             | ...                    |
+| `"UINT64"`               | `5121` <sup>3</sup>      | `"SINGLE"`      | `"SCALAR"`             |
+| ...                      | ...                      | ...             | ...                    |
+| `"INT8"`                 | `5120` (BYTE)            | `"SINGLE"`      | `"SCALAR"`             |
+| ...                      | ...                      | ...             | ...                    |
+| `"INT16"`                | `5122` (SHORT)           | `"SINGLE"`      | `"SCALAR"`             |
+| ...                      | ...                      | ...             | ...                    |
+| `"INT32"`                | `5124` (INT)<sup>4</sup> | `"SINGLE"`      | `"SCALAR"`             |
+| ...                      | ...                      | ...             | ...                    |
+| `"INT64"`                | `5121` <sup>3</sup>      | `"SINGLE"`      | `"SCALAR"`             |
+| ...                      | ...                      | ...             | ...                    |
+| `"FLOAT32"`              | `5126` (FLOAT)           | `"SINGLE"`      | `"SCALAR"`             |
+| ...                      | ...                      | ...             | ...                    |
+| `"FLOAT64"`              | `5121` <sup>3</sup>      | `"SINGLE"`      | `"SCALAR"`             |
+| ...                      | ...                      | ...             | ...                    |
+| `"BOOLEAN"`              | `5121` <sup>3</sup>      | `"SINGLE"`      | `"SCALAR"`             |
+| `"BOOLEAN"`              | `5121` <sup>3</sup>      | `"ARRAY"`       | `"SCALAR"`<sup>1</sup> |
+| `"STRING"`               | `5121` <sup>3</sup>      | `"SINGLE"`      | `"SCALAR"`             |
+| `"STRING"`               | `5121` <sup>3</sup>      | `"ARRAY"`       | `"SCALAR"`<sup>1</sup> |
+| `"ENUM"`                 | `5121` <sup>3</sup>      | `"SINGLE"`      | `"SCALAR"`             |
+| `"ENUM"`                 | `5121` <sup>3</sup>      | `"ARRAY"`       | `"SCALAR"`<sup>1</sup> |
+
+<small><sup>1</sup> Use "SCALAR" accessors for "ARRAY" property types, with `componentCount` for fixed-length arrays.</small><br>
+<small><sup>3</sup> `5125` (UNSIGNED_INT) disallowed by base glTF 2.0 specification except as `primitive.indices`. `EXT_mesh_features` overrides this restriction.</small><br>
+<small><sup>3</sup> `5121` (UNSIGNED_BYTE) used as general-purpose binary storage.</small><br>
+<small><sup>4</sup> `5124` (INT) disallowed by base glTF 2.0 specification. `EXT_mesh_features` overrides this restriction.</small>
```

Comment:
> `5124` (INT) disallowed by base glTF 2.0 specification. `EXT_mesh_features` overrides this restriction.

Should we also allow other GL types like `5130` (DOUBLE), `5134` (INT64), and `5135` (UINT64)?

```
#define GL_BYTE                           0x1400  (5120)
#define GL_UNSIGNED_BYTE                  0x1401  (5121)
#define GL_SHORT                          0x1402  (5122)
#define GL_UNSIGNED_SHORT                 0x1403  (5123)
#define GL_INT                            0x1404  (5124)
#define GL_UNSIGNED_INT                   0x1405  (5125)
#define GL_FLOAT                          0x1406  (5126)
#define GL_DOUBLE                         0x140A  (5130)
#define GL_INT64_ARB                      0x140E  (5134)
#define GL_UNSIGNED_INT64_ARB             0x140F  (5135)

Reply:
```suggestion
<small><sup>4</sup> `5124` (INT) is disallowed by the base glTF 2.0 specification. `EXT_mesh_features` overrides this restriction.</small>
```

Reply:
@lilleyse Oh I wasn't aware GL had enums for the 64-bit types. Since we're using int64 and uint64 already, I think it makes sense to use those GL enum values here. I'd probably omit `5130` (DOUBLE) since it hasn't come up before and isn't part of the core metadata spec.

Reply:
Added GL types for INT64 and UINT64.